.github/workflows/deploy.yml

```diff
@@ -0,0 +1,58 @@
+name: Build and deploy production
+
+on:
+  push:
+    branches:
+      - rezrah/azure-app-service-spike
+      # - main
```

Comment:
This will be uncommented before merging to ensure it only triggers the workflow in `main` branch.